package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "private-box",
   "description": "encrypt a message to a secret number of recipients",
-  "version": "0.1.1",
+  "version": "0.1.2",
```

Comment:
you shouldn't set the version number when making a PR. that is something the maintainer sets after merging. they may merge multiple PRs and then do one publish, so better to leave that to them.